pkg/bufferdecoder/net_decoder.go

```diff
@@ -8,7 +8,7 @@ import (
 
 type NetEventMetadata struct {
 	TimeStamp   uint64   `json:"timeStamp"`
-	NetEventId  uint32   `json:"netEventId"`
+	NetEventId  int32    `json:"netEventId"`
```

Comment:
Because now the type of the net events will be int32 (as the regular events type)
So if we just parse it at the first time as int32 , we will save later the cast in the `netEventsProcessor`